src/Controller/ComposterController.php

```diff
@@ -65,6 +65,33 @@ public function getCompostersGeojson(Request $request): Response
         return $this->json($geojson);
     }
 
+     /**
+     * @Route("/composters-cartoquartiers.json", name="composters-cartoquartiers")
+     * @param Request $request
+     * @return Response
+     */
+    public function getCompostersCartoQuartiers(Request $request): Response
+    {
+        $composters =  $this->getDoctrine()
+            ->getRepository(Composter::class)
+            ->findAllForCartoQuartierFrontMap();
+
+        $cartoQuartierfeatures = [];
+        /** @var Composter $c */
+        foreach ($composters as $c) {
+            $cartoQuartierfeatures[] = [
+                'IDOBJ'       => $c->getSerialNumber(),
+                'descriptif'  => $c->getPublicDescription(),
+                'photo'      => $c->getImage() ? $this->getImageUrl( $c->getImage()->getImageName() ) : null,
+                'horaires'    => $c->getPermanencesDescription(),
+                'mail'    => 'https://composteurs.compostri.fr/composteurs/'.$c->getSlug(),
```

Comment:
Désolé mais en front la route est au singulier ( et sans le U mais ca on va le changer en front )
```suggestion
                'mail'    => 'https://composteurs.compostri.fr/composteur/'.$c->getSlug(),
```